digitalstrategie/assets/scss/components/_faq_block.scss

```diff
@@ -0,0 +1,13 @@
+.accordion {
+    background-color: #fff;
+
+    &.yellow {
+        background-color: #f8ed45;
```

Comment:
You shouldn't use the colours in the componnets, but have a variable for the colours. @phillimorland suggested to replace pink, purple and yellow in the variables file (https://github.com/liqd/digitalstrategie/blob/20549143f4e76e397fa07400fd43d2b4166dd527/digitalstrategie/assets/scss/_variables.scss#L12) and use these here.